repository/a.json

```diff
@@ -690,6 +690,17 @@
 				}
 			]
 		},
+		{
+			"name": "AlpineJS",
+			"details": "https://github.com/SublimeText/AlpineJS",
+			"labels": ["syntax"],
```

Comment:
The most commonly used label for this would be [“language syntax”](https://packagecontrol.io/browse/labels/language%20syntax).